apps/api/src/services/data-provider/manual/manual.service.ts

```diff
@@ -166,11 +166,39 @@ export class ManualService implements DataProviderInterface {
         }
       });
 
-      for (const { currency, symbol } of symbolProfiles) {
+      const profilesWithScraping = symbolProfiles.filter(
+        ({ scraperConfiguration }) => {
+          return !!scraperConfiguration?.url;
+        }
+      );
+
+      const scrapingPromises = profilesWithScraping.map(
+        ({ symbol, scraperConfiguration }) => {
```

Comment:
`{ scraperConfiguration, symbol }`

Reply:
Hey, isn't this the same as { symbol, scraperConfiguration }, I made the changes just wanted to know why this way?

Reply:
The parameters are sorted alphabetically through out the project for consistency 🙂

Reply:
Oh, got it.